x-pack/plugins/aiops/public/components/spike_analysis_table/spike_analysis_table_groups.tsx

```diff
@@ -280,7 +280,7 @@ export const SpikeAnalysisGroupsTable: FC<SpikeAnalysisTableProps> = ({
                 +{Object.keys(repeatedValues).length}{' '}
                 <FormattedMessage
                   id="xpack.aiops.explainLogRateSpikes.spikeAnalysisTableGroups.moreLabel"
-                  defaultMessage="more"
+                  defaultMessage="more field names and values also appearing in other groups"
```

Comment:
>                   defaultMessage="more field names and values also appearing in other groups"

Since the tooltip just says "field values", can we do the same here? That is:

>                   defaultMessage="more field values also appearing in other groups"

Reply:
> @walterra says that we use `field/value pairs` in other parts of the UI, such as the loading state. So suggest we use that for both the badge text and the tooltip e.g.
> 
> ```
> +2 more field/value pairs also appearing in other groups
> ```
> 
> ```
> Displays field/value pairs unique to the group. Expand row to see all field/value pairs.
> ```

Updated to this copy and updated the screenshot in the description. Ready for a final look when you get a chance - cc @lcawl, @peteharverson 🙏